public/static/app/perf-bb/PerfBBRejectsCtrl.js

```diff
@@ -0,0 +1,385 @@
+mciModule.controller('PerfBBRejectsCtrl', function (
```

Comment:
This is more of a long-term comment and I don't it expect it addressed in this PR, but I'm wondering if there is anything we can do to reduce the amount of duplication between the PerfBB controllers.

Reply:
I think we need to pull functionality into factories (like RejectState here). 

There is a fair amount of duplication, but there are also subtle differences.

Reply:
@john-m-liu IMO no, the outliers are per test whereas you whitelist per task.

I think trying to visually represent these 2 different concepts in the same list might be difficult (although I'm not sure how well it is done at the moment anyway).